### DISCUSSION TITLE
2.0.3 is out!

### DISCUSSION THREAD

**Discussion by Gissio** [Gissio/radpro#158] [Announcements]
It is my pleasure to announce the release of Rad Pro 2.0.3, bringing several refinements:

* The conversion factor now extends from 25 to 12800 cpm/µSv/h, making Rad Pro compatible with more sensitive detectors like scintillation crystals.

* When using sensitive detectors with a conversion factor of 600 cpm/µSv/h or higher, Rad Pro now caps the pulse indication at 600 cpm/µSv/h to provide approximately one pulse per second at background levels of radiation. This does not impact the actual measurements.

* On devices with a color screen, the main screen has been slightly rearranged for improved clarity.

* This update also includes some minor performance improvements.

* Added the [Rad Pro simulator](https://gissio.github.io/radpro/simulator/).

**Comment by alfmck** [comment]
There is something wrong in FS-5000 radpro v2.0.2 calculating Instantaneous values:
![cpm-cps](https://github.com/user-attachments/assets/b7a2be54-2bcd-426d-a5ed-cb3034a978ca)
 Instantaneous averaging is set to 30sek. CPM is changing highly with no reason and CPS calculating values from CPS by hand from this table are different. I observed CPM values indicated in device screen and in the table - they are the same.

![2024-11-26_114424](https://github.com/user-attachments/assets/e0258b8a-ea4d-48d7-8aff-037341521c75)

**Reply by Gissio** [reply]
I don't follow what you mean.

**Reply by alfmck** [reply]
I guess that 10:52:58 -10:53:04 timeslots CPM values (M column) are not true - they cannot be calculated from CPS values, supplied in S column. I cannot realize that after 10:52:48 -10:53:01 timeslots, where CPS values were 0, CPM values in 10:52:58 -10:53:04 timeslots popped up from 25.3 to 34.0.

**Reply by Gissio** [reply]
Did you [RTFM](https://en.wikipedia.org/wiki/RTFM)?

The manual says that "the instantaneous rate is estimated by dividing the number of pulses within the instantaneous rate averaging period, minus one, by the time between the first and last pulse within that period".

In your case, the instantaneous rate averaging period is a 30-second sliding window. Since old data can fall out of the window, the instantaneous rate value can change even if there are no new pulses.

**Reply by alfmck** [reply]
Tried to calculate Instantaneous CPM values from mine data in excel:
[2024-11-26_114424.xlsx](https://github.com/user-attachments/files/17950592/2024-11-26_114424.xlsx).   I got one point CPM=34.29, so after 15 sec with CPS=0, getting high increasing of Instantaneous CPM values **is possible.** 
I don't realize how to use results of this algorithm in my FS-5000- maybe will realize sometime later. When seeing these values with Geigerlog I thought that my tube is faulty.
Anyway, thank You, Gissio for nice patch gdev_radpro.py for Geigerlog. Making changes in it I create own algorithms and see results on excellent Geigerlog platform.

**Reply by Gissio** [reply]
I suggest you get a good statistics book and study Poisson parameter estimation!
